easybuild/easyconfigs/m/mctc-lib/mctc-lib-0.3.1-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'mctc-lib'
+version = '0.3.1'
+
+homepage = 'https://grimme-lab.github.io/mctc-lib'
+description = """Common tool chain for working with molecular structure data in various
+applications. This library provides a unified way to perform operations on
+molecular structure data, like reading and writing to common geometry file
+formats."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['03dc8ccba37413da70e55a07cef8e8de53bce33f5bb52c1f8db5fec326abe083']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('json-fortran', '8.3.0'),
+]
+
+configopts = ['-DBUILD_SHARED_LIBS=ON', '-DBUILD_SHARED_LIBS=OFF']
+
+sanity_check_paths = {
+    'files': ['bin/mctc-convert', 'lib/libmctc-lib.%s' % SHLIB_EXT],
+    'dirs': ['include/%(name)s', 'lib/cmake', 'lib/pkgconfig'],
+}
+
+sanity_check_commands = ["mctc-convert --help"]
+
+# run suite of tests with ctest
+test_cmd = 'ctest'
+runtest = ''
```

Comment:
this can be simplified now (see https://github.com/easybuilders/easybuild-easyblocks/pull/2838)
```suggestion
# run suite of tests with ctest
runtest = True
```

Reply:
Thanks for finding that. I have corrected that.